src/renderer/screens/exchange/Swap2/Form/FormSummary/SectionRate.js

```diff
@@ -82,4 +90,4 @@ const SectionRate = ({ swapTransaction, provider }: Props) => {
   );
 };
 
-export default SectionRate;
```

Comment:
I feel wrapping this component in `React.memo` does more harm than good considering every time we change anything, the props change 🤔